app/views/accounts/_form.haml

```diff
@@ -1,7 +1,8 @@
-.row
-  .col-sm-8
-    = simple_form_for @user, url: account_path do |f|
-      = f.error_notification
+= simple_form_for @user, url: account_path do |f|
```

Comment:
This should all go under admin/user/edit, as an administrative function instead of on the account edit.